content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
+---
+
+{{< callout url=""https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
+Risk Based Entity Insights is in private beta.
+{{< /callout >}}
+
+## Overview
+
+[Cloud SIEM's Risk Based Entity Insights][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
+
+With Risk Based Entity Insights, you can:
+
+- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
+- Triage relevant items in bulk.
+- Take mitigation steps such as creating an automated workflow or creating a global case for an entity.
+
+## Explore Entities
+
+### Query and filter entities
+
+On the [Entities Explorer page][4], you can view all entities that have at least one signal.
+
+{{< img src="security/entities/entities-explorer.png" alt="A list of entities and their risk scores in the Entities Explorer" style="width:100%;" >}}
```

Comment:
can we use a screenshot that filters for severity and entity type instead? 
<img width="1496" alt="image" src="https://github.com/DataDog/documentation/assets/147530771/c8e8f969-bd9f-47d7-9455-c176e222fe13">